crypto/crypto.go

```diff
@@ -59,6 +60,17 @@ type Crypto struct {
 	config  Config
 }
 
+func (client *Crypto) CheckHealth() map[string]core.Health {
+	response := make(map[string]core.Health)
+	if client.storage == nil {
+		response[ModuleName] = core.Health{Status: core.HealthStatusDown, Details: "no storage"}
```

Comment:
is this also the case if it isn't used? That would be a bit weird

Reply:
There should always be a form of storage: fs, vault or external. So this is a case of "This should never happen ™️ ".

Reply:
It implies that this is possible, it is not, so remove the nil-check.